### PR TITLE
Apply PnP JS Guidance and No JQuery Dependancy

### PR DESCRIPTION
**Pull request by Laul0**
| Q               | A
| --------------- | ---
| Bug fix?        | no
| New feature?    | no
| New sample?      | no
| Related issues?  | --

#### What's in this Pull Request?
- Applying PnP Guidance Embedding JavaScript SharePoint (MDS)
- Remove JQuery dependancy (Full JavaScipt DOM)
- Tested on SharePoint 2013 On-Premise & Online

#### Guidance
- Remove JQuery dependancy (Full JavaScipt DOM)
- Tested on SharePoint 2013 On-Premise & Online

### CODE REVIEW

**Comment by Laul0** [comment]
Note : I forgot to mention that in this commit, the viewport support retina screens (iOS device mainly, by adding the Apple meta and change scale automatically) and after several tests and some feedback (on SharePoint 2013 On-Premise and Online), the performances are better.
 The tests on SharePoint 2016 On-Premise are will be performed as soon as possible

**Comment by VesaJuvonen** [comment]
Hi Laul0, looks good, but since this is highly used capability, we will need to carefully test the changes also on our side cross SharePoint versions. This means that getting this PR processed will take a while.

**Comment by Laul0** [comment]
Hi Vesa, thanks.
Don't hesitate for any question, support or review.

File: Core/OfficeDevPnP.Core/Resources/SP-Responsive-UI.js [hunk 1]
```diff
@@ -1,120 +1,156 @@
...
+            var linkSettingsSectionsLvl = settingsRoot.getElementsByClassName('ms-linksection-level1');
+            for (var i = 0; i < linkSettingsSectionsLvl.length; i++) {
+                var self = linkSettingsSectionsLvl[i];
+                var settingsDiv = document.createElement('div');
+                settingsDiv.className = 'pnp-settingsdiv';
+                settingsDiv.appendChild(self.getElementsByTagName('img')[0]);
+                settingsDiv.appendChild(self.getElementsByClassName('ms-linksection-textCell')[0]);
+                settingsRoot.appendChild(settingsDiv);
+            }
+            settingsRoot.removeChild(settingsRoot[0].getElementsByTagName('table')[0]);
```
Comment (Laul0) [inline Core/OfficeDevPnP.Core/Resources/SP-Responsive-UI.js]: This line throw an error : 
``` PowerShell
settingsRoot.removeChild(settingsRoot[0].getElementsByTagName('table')[0]);
```

Replace it by :

``` PowerShell
settingsRoot.removeChild(settingsRoot.getElementsByTagName('table')[0]);
```

**Comment by Laul0** [comment]
I'm working on a additional Responsive Behavior on SharePoint 2013. May I propose this one immediately or I have to waiting the merge of my current request first ?
All informations are available into the comment on the fork https://github.com/Laul0/PnP-Sites-Core/commit/5947e7c9b322e1e3c569a87973eb00f1e68cc306

**Comment by PaoloPia** [comment]
Hi Laul0,
I'm reviewing this PR but you removed the source branch, so I cannot do proper testing of the changes. Based on the fact that you are planning to add much more capabilities in the near future, do you mind to re-submit the PR (complete) and to keep the reference branch to make it possible for me to do proper testing? In the meantime I will close this PR, which looks good but cannot be easily tested. Thanks for your contribution.

**Comment by Laul0** [comment]
Hi Paolo, sorry for the inconvenience, please find the PR #951

**Pull request closed without merging**